docs/cheaha/tutorial/using_conda_project_space.md

```diff
@@ -0,0 +1,110 @@
+
+## Load the Miniforge3 Module
+
+Next, log in to Cheaha, and gain access to a Compute Node using the Terminal. There are detailed instructions [here](../getting_started.md#login-vs-compute-nodes). When in a compute node, load the Miniforge3 module to use Conda:
+
+```bash
+module load Miniforge3
+```
+
+![!Loading the Miniforge3 Module on a Cheaha Compute Node](images/module_load_miniforge.png)
+
+## Create a Conda folder in your Project Space on Cheaha
+
+**Navigate to the shared location where you want to create your Conda folder. Make sure that this location has the necessary permissions for all collaborators:**
+
+```bash
+cd /data/project/chosen/shared/location
+```
+
+![!Navigating to your project space on Cheaha](images/cheaha_project_space.png)
+
+**Create the Conda folder:**
+
+```bash
+mkdir conda_project
+
+cd conda_project
+```
+
+![!Creating a directory for your conda files](images/created_conda_directory.png)
+
+**You may need to adjust the permissions to ensure the folder is accessible by all members of your project space on Cheaha:**
+
+```bash
+chmod -R 770 conda_project
+```
+
+![!Granting all members of the project space access to the Conda directory](images/create_access_group.png)
+
+This command grants read, write, and execute permissions to the user and group (i.e. You as owner of the project space on Cheaha, as well as members of your project space), this will allow members of your project space on Cheaha access, and restrict others.
+
+## Initialize Conda
+
+To create a Conda environment that can be shared with members of your project space, we use the command below to initialize Conda in the folder location:
+
+```bash
+conda config --add envs_dirs /data/project/preferred/shared/location/conda_project/envs
+```
+
+![!Creating conda directories inside your folder](images/add_conda_config_directory.png)
+
+This command sets the Conda environment directory to the shared folder within your project space on Cheaha.
+
+## Create a Conda Environment in the Shared Folder
+
+Now, create a new Conda environment in the shared project folder:
+
+```bash
+conda create --prefix /data/project/preferred/shared/location/conda_project/envs/my_env python=3.11
+```
+
+![!Command showing the creation of A Conda env with the prefix flag](images/conda_prefix_env.png)
+
+Replace `/data/project/preferred/shared/location/conda_project/envs/my_env` with the path to your shared folder and `python=3.11` with the desired version of Python or any other package you want to install. This will create a Conda environment named `my_env`. This can be replaced with an environment name of your choosing.
+
+Activate the environment:
+
+```bash
+conda activate /data/project/chosen/shared/location/conda_project/envs/my_env
```

Comment:
Have you seen this page? The recommendation here is like what you have, but with the --prefix flag. Interesting to know it works without the flag!

Testing it myself, I see it only works without the flag. Looks like the official docs has an error.